### PR TITLE
Add osu! Indonesia Tournament #11 (2021) wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
wiki article for the currently ongoing osu! Indonesia Tournament #11 (2021)

### CODE REVIEW

File: wiki/Tournaments/OIT/2021/en.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+| Graphic designer | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805), ![][flag_ID] [Tama-desu](https://osu.ppy.sh/users/3813174), ![][flag_ID] [Firia](https://osu.ppy.sh/users/9730262), ![][flag_FI] [shdewz](https://osu.ppy.sh/users/10000899) |
+| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |
+
+## Links
+
+- **[Master spreadsheet](https://docs.google.com/spreadsheets/d/1R7y585_PIb5XrFvDQk-rlI0vSlr2FyGkhCmPzorNhuU/edit#gid=1680086225)**
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1215072)
+- [Discord server](https://discordapp.com/invite/CHRFffH)
+- [Livestream channel](https://www.twitch.tv/osuindonesia)
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:56]:
```suggestion
- **[Master spreadsheet](https://docs.google.com/spreadsheets/d/1R7y585_PIb5XrFvDQk-rlI0vSlr2FyGkhCmPzorNhuU/edit#gid=1680086225 "Google Sheets")**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1215072)
- [Discord server](https://discordapp.com/invite/CHRFffH "Discord")
- [Livestream channel](https://www.twitch.tv/osuindonesia "Twitch")
```

File: wiki/Tournaments/OIT/2021/en.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  - [Hanatan - Kokoro (sodarose) \[Miracle\]](https://osu.ppy.sh/beatmapsets/1005463#osu/2104338)
+  - [Sunpon - Skyscraper Grief (Natteke) \[Happy's Lunatic\]](https://osu.ppy.sh/beatmapsets/15200#osu/69390)
+
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
+2. The mapsets for each round will be announced by the Tournament Management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the Tournament Management. If there are any player(s) who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the #scheduling channel in the tournament's Discord server.
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:85]:
```suggestion
3. Match schedules will be predetermined by the Tournament Management. If there are any player(s) who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the `#scheduling` channel in the tournament's Discord server.
```

File: wiki/Tournaments/OIT/2021/en.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+8. If a player disconnects, it will be treated as if they failed the beatmap.
+   - Disconnects that occur within a few seconds after the beatmap is started by the referee can be rematched.
+9. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes at most.
+   - In case the disconnected player failed to report back after the maximum allowance of 15 minutes had passed, the opposing player may be declared to be winning the match by default.
+10. Lag is not a valid reason to nullify a beatmap.
+11. If there are any problems during the match occurence, the Tournament Management will make a decision based on referee's report.
+12. It is expected that all players be polite and respectful to each other. Penalties will be given if players violate.
+    - If a player is found to be engaging in an act of offensive provocation, the said player responsible for the provocation may be either disqualified right away from OIT 2021 and/or blacklisted from the future installment(s) of OIT.
+    - Usage of any illegal programs is strictly prohibited and will be straight up reported to the osu!staff at will.
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:98]: - to me, "illegal" is something that is forbidden by the law. should probably reword this to underline that it breaks osu!'s rules instead
- `osu!staff` -> `osu!team`

File: wiki/Tournaments/OIT/2021/en.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    - If a player is found to be smurfing another player's account, the corresponding players will be blacklisted in the next installment(s) of OIT regardless of their role in the smurfing accident.
+
+### Qualifier rules
+
+1. Each player will have to sign up to one of the twenty Qualifier lobbies that has been scheduled and prepared by the Tournament Management in advance.
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:103]:
```suggestion
1. Each player will have to sign up to one of the twenty Qualifier lobbies that have been scheduled and prepared by the Tournament Management in advance.
```

File: wiki/Tournaments/OIT/2021/en.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+2. In the lobby, players will have to consecutively play all of the ten Qualifier beatmaps in the order of NM1 -> NM2 -> NM3 -> NM4 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2.
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:104]: a minor nit: I try to cater to typographics where possible (including the usage of symbolic arrows)

File: wiki/Tournaments/OIT/2021/en.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+2. Players will compete against each other using the Double Elimination system.
+3. The Double Elimination System works as follows:
+   - Players who lose in the Upper Bracket can still play again on the Lower Bracket.
+   - Players who lose in the Lower Bracket will be eliminated from the tournament.
+   - In the Grand Finals, the winner of the the Upper Bracket will only need to win a single match against their opponent in order to claim the championship title. The winner of the Lower Bracket, however, will need to win two matches and enforce a Bracket Reset against their opponent in order to claim the championship title.
+4. Players who can compete in the next round are determined by:
+   - In the Round of 32 and the Round of 16, each player needs to win 5 points in order to win a match. (Best-of-9)
+   - In the Quarterfinals and the Semifinals, each player needs to win 6 points in order to win a match. (Best-of-11)
+   - In both of the Finals Weeks, each player needs to win 7 points in order to to win a match. (Best-of-13)
+   - Players who Win By Default.
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:123]: is the capping necessary here? I get that `win by default` is a specific term, but here it fits better as a part of the sentence

File: wiki/Tournaments/OIT/2021/en.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+   - The loser of the `!roll` gets the opposite by default.
+   - This rule does not apply in the Qualifier lobbies.
+2. Each player has to ban **two beatmaps** from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
+   - Each player may not ban two beatmaps from the same mod pool, except from the NoMod pool. For example, banning NM1 and NM2 is allowed; banning NM1 and HD1 is allowed; banning HD1 and HD2, however, is *not* allowed.
+   - Banning does not apply in the Qualifier lobbies.
+3. Each player is free to select one warm-up beatmap. Using beatmaps with questionable content is prohibited.
+   - Warm-ups do not apply in the Qualifier lobbies.
+4. In a FreeMod pick, each player has to apply at least one mod to play the beatmap with. Allowed mods are EZ, HR, HD, or any possible combinations of the three mods.
+   - Playing a FreeMod pick without any mods applied is not allowed.
+5. The results of each match and any other relevant information regarding the match will be posted on the Discord server after the match has concluded by the responsible referees.
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:139]:
```suggestion
5. The results of each match and any other relevant information regarding the match will be posted on the Discord server after the match has been concluded by the responsible referees.
```

File: wiki/Tournaments/OIT/2021/en.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+[flag_FI]: /wiki/shared/flag/FI.gif "Finland"
```
Comment (TicClick) [inline wiki/Tournaments/OIT/2021/en.md:141]: lone shdewz

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/OIT/2021/en.md (hunk 8 above)
Comment (Nivalyx) [reply wiki/Tournaments/OIT/2021/en.md:141]: 🇫🇮

**Review by TicClick — APPROVED** [review]
